.github/workflows/refapp-2x-clinical-visit.yml

```diff
@@ -47,5 +47,5 @@ jobs:
           echo "cucumber.publish.enabled=true" > src/test/resources/cucumber.properties
           mvn clean install -DskipTests=true
           cp -f src/test/resources/org/openmrs/uitestframework/test-local.properties src/test/resources/org/openmrs/uitestframework/test.properties
-          npm run clinicalVisitTest
+          npm run refapp2ClinicalVisitTest
```

Comment:
Can we make it refApp_2.xClinicalVisitTest or something case sensitive.

Reply:
> Can we make it refApp_2.xClinicalVisitTest or something case sensitive. with other defined hook


I took a look at the exisiting naming convention eg for [refappSelenium](https://github.com/openmrs/openmrs-contrib-qaframework/blob/0c23df71ec10866a4c29f633444f6d8236f41500/.github/workflows/qa.yml#L50) [platformUpgrade](https://github.com/openmrs/openmrs-contrib-qaframework/blob/0c23df71ec10866a4c29f633444f6d8236f41500/.github/workflows/platform-upgrade.yml#L60) [simpleCoreInstall](https://github.com/openmrs/openmrs-contrib-qaframework/blob/0c23df71ec10866a4c29f633444f6d8236f41500/.github/workflows/platform-install-simple.yml#L59) they shouts out clearly that they actually accomplish.

I suggested we would rather have our new test workflows for refapp 2.x be renamed to such names have proposed. They shouts out clearly well like `refapp2ClinicalVisitTest` mentions clearly the` refapp` version and what the test does. I was hesitant not to put the `x` value on the integer version value since it caters for all 2.x versions so I would like to think we embrace the suggested one.

Reply:
> refapp2ClinicalVisitTest

refApp2.xClinicalVisitTest , what do you think about that naming convention.

Reply:
> **refApp2.xClinicalVisitTest ,** what do you think about that naming convention.

I have given the reason [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/69#discussion_r687751440) why I do think we should not attach `x` value on the integer version value.

Reply:
> i think we need to have **x** on the hook to make it more clear.

Actually this is not a hook but a command that triggers a workflow, based on the reason have given [above ](https://github.com/openmrs/openmrs-contrib-qaframework/pull/69#discussion_r687751440) concerning why we should not attach the `x` value its sufficient enough not to attach `x` value on an the integer version `value`.